src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -119,19 +119,20 @@
   </resheader>
   <data name="FirstTimeWelcomeMessage" xml:space="preserve">
     <value>Welcome to .NET Core!
----------------------
-Learn more about .NET Core @ https://aka.ms/dotnet-docs. Use dotnet --help to see available commands or go to https://aka.ms/dotnet-cli-docs.
+----------------------------------
+Learn more about .NET Core: https://aka.ms/dotnet-docs
+Use 'dotnet --help' to see available commands or see: https://aka.ms/dotnet-cli-docs
 
 Telemetry
---------------
-The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community.
-You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell.
-You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.</value>
+----------------------------------
+The .NET Core tools collect usage data in order help us improve your experience. The data is anonymous and doesn't include command-line arguments. The data is collected by Microsoft and shared with the community. Opt-out of telemetry by setting the DOTNET_CLI_TELEMETRY_OPTOUT environment variable to '1' or 'true' using your favorite shell.
```

Comment:
It's written from the perspective of the company, so MS's tools collect user data in order to help us (Microsoft) improve your (the user's) experience. Of course, we could try some other clauses/phrases.